inscriptions/ISic000017.xml

```diff
@@ -198,16 +198,16 @@
-                    <lb n="6"/><del rend="erasure"><supplied reason="lost"><expan><abbr>fil</abbr><ex>io</ex></expan></supplied></del> et <g ref="#interpunct">·</g> <expan><abbr>imp</abbr><ex>eratoris</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>Caes</abbr><ex>aris</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>M</abbr><ex>arci</ex></expan> <g ref="#interpunct">·</g> Aureli <g ref="#interpunct">·</g> An
-                    <lb n="7" break="no"/>tonini <g ref="#interpunct">·</g> <expan><abbr>Aug</abbr><ex>usti</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>trib</abbr><ex>unicia</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>pot</abbr><ex>estate</ex></expan> <g ref="#interpunct">·</g> <del rend="erasure"><supplied reason="lost">fratri</supplied></del>
-                    <lb n="8"/>domino <g ref="#interpunct">·</g> indulgentissimo
+                    <lb n="4" break="no"/>aben</abbr><ex>ici</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>Parth</abbr><ex>ici</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>m<unclear>ax</unclear></abbr><ex>imi</ex></expan> <g ref="#interpunct">·</g> <roleName type="supracivic.religious" subtype="pontifex.maximus"><expan><abbr>pont</abbr><ex>ificis</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>m<supplied reason="lost">ax</supplied></abbr><supplied reason="lost"><ex>imi</ex></supplied></expan></roleName>  <roleName type="supracivic" subtype="tribunicia.potestas"><expan><abbr>tr</abbr><ex>ibunicia</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>pot</abbr><ex>estate</ex></expan></roleName> <g ref="#interpunct">·</g> <num value="6">VI</num> <g ref="#interpunct">·</g> <roleName type="supracivic" subtype="imperator"><expan><abbr>imp</abbr><ex>eratori</ex></expan></roleName>
+                    <lb n="5"/><roleName type="supracivic" subtype="tribunicia.potestas"><expan><abbr>tr</abbr><ex>ibunicia</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>pot</abbr><ex>estate</ex></expan></roleName> <g ref="#interpunct">·</g> <num value="6">VI</num> <g ref="#interpunct">·</g> <roleName type="supracivic" subtype="imperator"><expan><abbr>imp</abbr><ex>eratori</ex></expan></roleName> <num value="11">XI</num> <roleName type="supracivic" subtype="consul"><expan><abbr>co</abbr><ex>n</ex><abbr>s</abbr><ex>ulis</ex></expan></roleName> <g ref="#interpunct">·</g> <num value="2">II</num> <g ref="#interpunct">·</g> <supplied reason="lost"><expan><abbr>p</abbr><ex>atri</ex></expan></supplied> <supplied reason="lost"><g ref="#interpunct">·</g></supplied> <roleName type="supracivic" subtype="pater.patriae"><supplied reason="lost"><expan><abbr>p</abbr><ex>atriae</ex></expan></supplied> </roleName>
```

Comment:
tag around pater patriae has included patriae only, and needs to include pater also.